Change hanglm24.bdf to gulim24.bdf as this is the one we have.

---
 lisp/ps-mule.el |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- lisp/ps-mule.el
+++ lisp/ps-mule.el	2016-09-19 08:57:28.807664990 +0000
@@ -262,7 +262,7 @@ Currently, data for Japanese and Korean
     (japanese-jisx0208
      (normal bdf "jiskan24.bdf"))
     (korean-ksc5601
-     (normal bdf "hanglm24.bdf"))
+     (normal bdf "gulim24.bdf"))
     (japanese-jisx0212
      (normal bdf ("jksp40.bdf" "jisksp40.bdf")))
     (chinese-cns11643-1
